File: nivel-04/ex-01/carregaVetor.js
var assert = require('assert');

var vetor = [4, 1, 6, 9, 2]

try {

    assert.equal(4, vetor[0]);
    assert.equal(1, vetor[1]);
    assert.equal(6, vetor[2]);
    assert.equal(9, vetor[3]);
    assert.equal(2, vetor[4]);

} catch(e) {
    console.log(e);
}

File: nivel-04/ex-01/carregaVetor.c
// Código que carrega vetor

#include <stdio.h>
#include <assert.h>

void main(){
	int vetor[5];

	for (int i = 0; i < 5; i++){
		
		vetor[i] = i + 1; 
	}
	
	assert(1 == vetor[0]);
	assert(2 == vetor[1]);
	assert(3 == vetor[2]);
	assert(4 == vetor[3]);
	assert(5 == vetor[4]);
}
